Woof/Woof/Helper/AppButton/AppButton.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+/// Represents name of buttons which uses in application.
+enum AppButton: String, CustomStringConvertible {
+    case cancel = "Cancel"
+    case save = "Save"
+    case edit = "Edit"
+    case tryAgain = "Try Again"
+    case logout = "Logout"
+    case `continue` = "Continue"
```

Comment:
Missing doc comments, as mentioned above let's create separate PR for this as well

Reply:
Fixed with 19ade1f